Mathlib/Analysis/SpecialFunctions/Integrals.lean

```diff
@@ -741,6 +741,19 @@ theorem integral_cos_sq : ∫ x in a..b, cos x ^ 2 = (cos b * sin b - cos a * si
   by field_simp [integral_cos_pow, add_sub_assoc]
 #align integral_cos_sq integral_cos_sq
 
+theorem integral_sqrt_one_sub_sq : ∫ x in (-1 : ℝ)..1, sqrt (↑1 - x ^ 2) = π / 2 :=
+  calc
+    _ = ∫ x in (-(π / 2))..(π / 2), sqrt (↑1 - sin x ^ 2) * cos x := by
+          rw [show ↑(-1) = sin (-(π / 2)) by { rw [sin_neg, sin_pi_div_two] }]
+          nth_rewrite 2 [← sin_pi_div_two]
+          exact (integral_comp_mul_deriv (fun x _ => hasDerivAt_sin x) continuousOn_cos
+            (by continuity)).symm
```

Comment:
I think this is somewhat more readable in two steps:
```suggestion
    _ = ∫ x in sin (-(π / 2))..sin (π / 2), sqrt (↑1 - x ^ 2) := by
          rw [sin_neg, sin_pi_div_two]
    _ = ∫ x in (-(π / 2))..(π / 2), sqrt (↑1 - sin x ^ 2) * cos x :=
          (integral_comp_mul_deriv (fun x _ => hasDerivAt_sin x) continuousOn_cos
            (by continuity)).symm
```

Reply:
Yes. you're right. Thanks!